students/AntonBakaiev/hw-1/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">  
+    <link rel="stylesheet" href="style.css">
+    <script>
```

Comment:
One of the good practices is not to include scripts in the `head` section of a document. Better place to it at the bottom of the `body` section.

Reply:
fixed